packages/sdk-react/src/components/payables/PayablesTable/consts.ts

```diff
@@ -4,7 +4,7 @@ export const FILTER_TYPE_SEARCH = 'search';
 export const FILTER_TYPE_STATUS = 'status';
 export const FILTER_TYPE_DUE_DATE = 'due_date';
 export const FILTER_TYPE_CREATED_AT = 'created_at';
-export const FILTER_TYPE_OVERDUE = 'overdue';
+export const FILTER_TYPE_CUSTOM_MONITE = 'custom_monite';
```

Comment:
It was difficult to understand the purpose of this filter by name. Let's use something more descriptive. From what I understood it should be something like `summary_card_filter`